docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN cargo build --locked --profile $PROFILE $BUILD_ARGS
 # ==========================
 # stage 2: packaging
 # ==========================
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
We can update this only when our production parachain nodes are all deployed with docker.